libs/s25main/ingameWindows/iwStatistics.cpp

```diff
@@ -15,10 +15,25 @@
 #include "iwHelp.h"
 #include "network/GameClient.h"
 #include "ogl/FontStyle.h"
+#include "ogl/glFont.h"
 #include "world/GameWorldBase.h"
 #include "world/GameWorldViewer.h"
 #include "gameData/const_gui_ids.h"
 
+namespace {
+
+std::string getPlayerStatus(const GamePlayer& player)
+{
+    if(player.IsDefeated())
+        return "---";
+    else if(player.isHuman())
+        return "#" + std::to_string(player.GetPlayerId() + 1);
+    else
+        return _("COMP");
```

Comment:
Is this in all languages COMP? Checked english and it is, but I'm not that sure about it

Reply:
It must be translated because in france it's ordinateur.